### ISSUE TITLE
promise-7.0.4.min.js.map is not found

### ISSUE THREAD

**Issue by israelor6** [mapsplugin/cordova-plugin-googlemaps#2456]
**I'm submitting a ...**  (check one with "x")
- [ ] question
- [x] any problem or bug report

**OS:**  (check one with "x")
- [ ] **Android**
- [x] **iOS**
- [ ] **Browser**

**cordova information:**  (run `$> cordova plugin list`) 

```
com.pylonproducts.wifiwizard 0.2.11 "WifiWizard"
cordova-android-support-gradle-release 1.4.4 "cordova-android-support-gradle-release"
cordova-open-native-settings 1.5.1 "Native settings"
cordova-plugin-camera 4.0.3 "Camera"
cordova-plugin-crop 0.3.1 "CropPlugin"
cordova-plugin-device 2.0.2 "Device"
cordova-plugin-file 6.0.1 "File"
cordova-plugin-filechooser 1.0.1 "File Chooser"
cordova-plugin-filepath 1.4.2 "cordova-plugin-filepath"
cordova-plugin-geolocation 4.0.1 "Geolocation"
cordova-plugin-googlemaps 2.4.4 "cordova-plugin-googlemaps"
cordova-plugin-ionic-webview 1.2.1 "cordova-plugin-ionic-webview"
cordova-plugin-networkinterface 2.0.0 "NetworkInterface"
cordova-plugin-splashscreen 5.0.2 "Splashscreen"
cordova-plugin-statusbar 2.4.2 "StatusBar"
cordova-plugin-whitelist 1.3.3 "Whitelist"
cordova-sqlite-storage 2.4.0 "Cordova sqlite storage plugin"
cordova-wheel-selector-plugin 1.0.0 "Cordova Wheel Selector Plugin"
ionic-plugin-keyboard 2.2.1 "Keyboard"
```

**If you use `@ionic-native/google-maps`, please tell the package.json (only `@ionic-native/core` and `@ionic-native/google-maps` are fine mostly)**
```
@ionic-native/core : ^4.15.0
@ionic-native/google-maps : ^4.14.0
```

**Current behavior:**

i got promise-7.0.4.min.js.map is not found error.
**Expected behavior:**

**Screen capture or video record:**

**Related code, data or error log (please format your code or data):**

```
Please share your project files on Github or others(Bitbucket, Gitlabs...etc).
If you don't want to share your project files, please create a demo project, then share it.

Screen captures, and/or native logs(such as Logcat, xcode logs) are appreciate.

Giving much information, you are waiting time is less.
Thank you for your cooperation.
```

**Comment by wf9a5m75** [comment]
It's not fatal problem. You just ignore it.
If you don't want to ignore it, please reinstall the plugin from the multiple_maps branch.

**Comment by israelor6** [comment]
in browser (cordova run browser -l) its works fine.

**Comment by francovp** [comment]
This was supposed to be fixed on 2.4.5, but I'm experienced the same error on 2.4.6
